### PR TITLE
fix: export const

### PR DESCRIPTION
**Pull request by sahar-fehri**
## Explanation

PR to export `SUPPORTED_CHAIN_IDS` constant.
This constant is imported in extension like this:
`import { SUPPORTED_CHAIN_IDS } from '@metamask/assets-controllers/dist/token-prices-service/codefi-v2';`
Which was causing issues when trying to upgrade assets-controllers to latest. It was not able to find it.
With this change we can import it like this instead of from dist which can abstract how the code is transpiled;
`import { SUPPORTED_CHAIN_IDS } from '@metamask/assets-controllers';`

## References

* Related to [#67890](https://github.com/MetaMask/metamask-extension/pull/23574)

### `@metamask/assets-controllers`

- **ADDED**: Added export for const SUPPORTED_CHAIN_IDS 

## Checklist

- [ ] I've updated the test suite for new or updated code as appropriate
- [ ] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [ ] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.test.ts [hunk 1]
```diff
@@ -48,4 +48,21 @@ describe('ERC1155Standard', () => {
       );
     expect(contractSupportsUri).toBe(true);
   });
+
+  it('should determine if contract supports token receiver interface correctly', async () => {
```
Comment (Gudahtt) [inline packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.test.ts:52]: This seems to be unrelated to the rest of the PR

Comment (sahar-fehri) [reply packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.test.ts:52]: Its not related but the coverage went lower for some reason and the CI was failing, so i added this test just to update the coverage

Comment (Gudahtt) [reply packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.test.ts:52]: Oh I see, that's because we don't have coverage for our package exports, but a new package export was added.

We could address this by adding a snapshot test for package exports, similar to what we have already for the token price service module
